packit_service/worker/handlers/distgit.py

```diff
@@ -409,9 +409,13 @@ def __init__(
     @property
     def pull_request(self):
         if not self._pull_request and self.data.event_dict["committer"] == "pagure":
+            logger.debug(
```

Comment:
```suggestion
            logger.info(
```

Reply:
I think we can not see debug messages in production and this seems not to be happening in stage. For this reason I will put it as an info message.

Reply:
We should see the debug messages both in prod and staging.